checks/microbenchmarks/cpu_gpu/node_burn/node-burn-ce.py

```diff
@@ -21,9 +21,16 @@ class NodeBurnCE(rfm.RunOnlyRegressionTest, ContainerEngineMixin):
     nb_duration = variable(int, value=20)
     nb_matrix_size = variable(int, value=40000)
     executable = 'burn-f64'
+    flexible = variable(bool, value=False)
```

Comment:
```suggestion
    flexible = variable(bool, value=False)
    time_limit = '2m'
```

Reply:
maintainers WS ?

Reply:
@jgphpc I want to live the `time_limit` as is because sometimes it takes more time to download the container image.